webroot/results/includes/statistics/most_solves_in_one_competition_or_year.php

```diff
@@ -1,5 +1,24 @@
 <?php
 
+$totalsolves = polishMostSolvesAttempts( "
+  SELECT    personId,
+            competitionId whereId,
```

Comment:
Not your fault, but the the variable names in here make no sense to me. There's `$totalsolves`, `$solves`, and `$attempts`. From reading, it looks like all 3 of these variables include both a number of solves *and* a number of a attempts, it's just either by career, competition, or year. If I'm understanding this correctly, would you change the three variables to something like `$mostSolvesInACareer`, `$mostSolvesInACompetition`, `$mostSolvesInAYear`?